### PR TITLE
[LENS-1324] Adds examples on setting new deviceId

### PR DESCRIPTION
**Pull request by npushkarskii**
This PR adds examples on how to set a new `deviceId`.
README already had the examples on how to get the current `deviceId` and `trackingId`.
https://coveord.atlassian.net/browse/LENS-1324

### CODE REVIEW

**Review by jsonDaCruzo — APPROVED** [review]
Thanks @npushkarskii, please wait for @hporter approval as he is a better judge on the code samples provided.

**Review by hporter — APPROVED** [review]

**Pull request merged**
